index.html

```diff
@@ -385,7 +385,7 @@ <h2><dfn>observe()</dfn> method</h2>
             <a>entryTypes</a> sequence.
             </li>
             <li>Remove all types from <var>entry types</var> that are not
-            contained in <a>supported entry types</a>. The user agent SHOULD
+            contained in <a>supported entry types</a> for the relevant <a>global object</a>. The user agent SHOULD
```

Comment:
We need to specify relevant with respect to what object. Here, we need to decide whether we want to get the global object with respect to the observer, context object, or something else.